cli/command/commands/commands.go

```diff
@@ -72,13 +72,19 @@ func AddCommands(cmd *cobra.Command, dockerCli *command.DockerCli) {
 		hide(system.NewInspectCommand(dockerCli)),
 	)
 
+	if versions.GreaterThanOrEqualTo(dockerCli.Client().ClientVersion(), "1.25") {
+		cmd.AddCommand(system.NewVersionCommand(dockerCli))
+	}
```

Comment:
Sorry, I tought "system." was for the new data management commands, I'll remove that.